python/ray/includes/libcoreworker.pxi

```diff
@@ -1,10 +1,27 @@
 from libcpp.string cimport string as c_string
 
 from ray.includes.libcoreworker cimport CProfileEvent
+from ray.includes.libcoreworker cimport CCoreWorker
+from ray.includes.unique_ids cimport CNodeID
+from ray.includes.unique_ids cimport NodeID
 
 import json
 import traceback
 
+
+# We cannot call this CoreWorker because _raylet.pxd defines an unrelated
+# cdef class CoreWorker and _raylet.pyx does
+# include "includes/libcoreworker.pxi"
+cdef class CoreWorkerWrapper:
+    """Cython wrapper class of C++ `ray::CoreWorker`."""
+    cdef:
+        unique_ptr[CCoreWorker] inner
+
+    def GetCurrentNodeId(self):
+        cdef CNodeID c_nodeID = inner.get().GetCurrentNodeId()
+        return NodeID(c_nodeID.Binary())
```

Comment:
The `CoreWorker` defined in _raylet.pyx is actually wrapping the same C++ class. You can just add this as a method to that.

Reply:
> The `CoreWorker` defined in _raylet.pyx is actually wrapping the same C++ class. You can just add this as a method to that.

That `CoreWorker` doesn't seem to have a `CCoreWorker inner`. I'd have to add one. Would that be all right?